pkg/scannerv4/mappers/mappers.go

```diff
@@ -695,6 +767,49 @@ func pkgFixedBy(enrichments map[string][]json.RawMessage) (map[string]string, er
 	return pkgFixedBys, nil
 }
 
+// cveEPSS unmarshals and returns the EPSS enrichment, if it exists.
+func cveEPSS(ctx context.Context, enrichments map[string][]json.RawMessage) (map[string]map[string]*epss.EPSSItem, error) {
+	if !features.EPSSScore.Enabled() {
+		return nil, nil
+	}
+	enrichmentList := enrichments[epss.Type]
+	if len(enrichmentList) == 0 {
+		zlog.Warn(ctx).
+			Str("enrichments", epss.Type).
+			Msg("no EPSS enrichments found")
+		return nil, nil
+	}
+
+	var epssItems map[string][]epss.EPSSItem
+	// The EPSS enrichment always contains only one element.
+	err := json.Unmarshal(enrichmentList[0], &epssItems)
+	if err != nil {
+		return nil, fmt.Errorf("unmarshaling EPSS enrichment: %w", err)
+	}
+
+	if len(epssItems) == 0 {
+		return nil, nil
+	}
+
+	ret := make(map[string]map[string]*epss.EPSSItem)
+	for ccVulnID, list := range epssItems {
+		if len(list) > 0 {
+			m := make(map[string]*epss.EPSSItem)
+			for idx := range list {
+				epssData := list[idx]
+				m[epssData.CVE] = &epss.EPSSItem{
+					CVE:          epssData.CVE,
+					Percentile:   epssData.Percentile,
+					EPSS:         epssData.EPSS,
+					ModelVersion: epssData.ModelVersion,
+				}
+			}
```

Comment:
No reason to not just reuse the pre-existing variable. Also the date was missing

```suggestion
				epssData := list[idx]
				m[epssData.CVE] = &epssData
			}
```